SU2_PY/discrete_adjoint.py

```diff
@@ -101,7 +106,7 @@ def discrete_adjoint( filename           ,
     konfig = copy.deepcopy(config)
 
     # Direct Solution
-    if compute:
+    if compute and not onlyadjoint:
```

Comment:
It seems like these two options should be incompatible. I see three exclusive modes: compute primal & adjoint & gradient, compute adjoint (with primal restart) & gradient, or compute gradient only (with primal and adjoint restarts.

Can you please add an error check or combine these into a single MODE option, perhaps?

Reply:
I've implemented your suggestions :)